multicluster/cmd/link.go

```diff
@@ -247,7 +247,7 @@ A full list of configurable values can be found at https://github.com/linkerd/li
 				if len(gwAddresses) == 0 && opts.gatewayAddresses == "" {
```

Comment:
an alternative way to structure this for clarity:

```
if opts.gatewayAddresses != "" {
  gatewayAddresses = opts.gatewayAddresses
} else if len(gwAddresses) > 0 {
  gatewayAddresses = strings.Join(gwAddresses, ",")
} else {
  return fmt.Errorf(...)
}
```

This condenses the error handling and gateway population into the same if-else block and makes the prescience clearer: opt then LB then error.